Makefile

```diff
@@ -612,7 +614,7 @@ cpplint:
 
 lint: jslint cpplint
 
-.PHONY: lint cpplint jslint bench clean docopen docclean doc dist distclean \
+.PHONY: lint cpplint jslint bench clean docopen docclean doc doc-only dist distclean \
```

Comment:
@jbergstroem Thanks, updated. FWIW there are some other [long lines](https://github.com/nodejs/node/blob/b801e39266b815baaf264167753bdc0bb7394489/Makefile#L70) with embedded whitespace in the file.